identifiers/country-in/federal_states_territories.csv

```diff
@@ -1,169 +1,704 @@
-ocd-division/country:in/state:jh/district:gu,Gumla
-ocd-division/country:in/state:jh/district:ha,Hazaribagh
-ocd-division/country:in/state:jh/district:ja,Jamtara
-ocd-division/country:in/state:jh/district:kh,Khunti
-ocd-division/country:in/state:jh/district:ko,Kodarma
-ocd-division/country:in/state:jh/district:la,Latehar
-ocd-division/country:in/state:jh/district:lo,Lohardaga
-ocd-division/country:in/state:jh/district:pk,Pakur
-ocd-division/country:in/state:jh/district:pl,Palamu
-ocd-division/country:in/state:jh/district:rm,Ramgarh
-ocd-division/country:in/state:jh/district:ra,Ranchi
-ocd-division/country:in/state:jh/district:sa,Sahebganj
-ocd-division/country:in/state:jh/district:sk,Saraikela Kharswan
-ocd-division/country:in/state:jh/district:si,Simdega
-ocd-division/country:in/state:jh/district:ws,West Singhbhum
-ocd-division/country:in/state:mh/district:ah,Ahmednagar
-ocd-division/country:in/state:mh/district:ak,Akola
-ocd-division/country:in/state:mh/district:am,Amaravati
-ocd-division/country:in/state:mh/district:au,Aurangabad
-ocd-division/country:in/state:mh/district:bi,Beed
-ocd-division/country:in/state:mh/district:bh,Bhandara
-ocd-division/country:in/state:mh/district:bu,Buldhana
-ocd-division/country:in/state:mh/district:ch,Chandrapur
-ocd-division/country:in/state:mh/district:dh,Dhule
-ocd-division/country:in/state:mh/district:ga,Gadchiroli
-ocd-division/country:in/state:mh/district:go,Gondiya
-ocd-division/country:in/state:mh/district:hi,Hingoli
-ocd-division/country:in/state:mh/district:jg,Jalgaon
-ocd-division/country:in/state:mh/district:jn,Jalna
-ocd-division/country:in/state:mh/district:ko,Kolhapur
-ocd-division/country:in/state:mh/district:la,Latur
-ocd-division/country:in/state:mh/district:mc,Mumbai City
-ocd-division/country:in/state:mh/district:mu,Mumbai Suburban
-ocd-division/country:in/state:mh/district:ng,Nagpur
-ocd-division/country:in/state:mh/district:nd,Nanded
-ocd-division/country:in/state:mh/district:nb,Nandurabar
-ocd-division/country:in/state:mh/district:ns,Nashik
-ocd-division/country:in/state:mh/district:os,Osmanabad
-ocd-division/country:in/state:mh/district:pl,Palghar
-ocd-division/country:in/state:mh/district:pa,Parbhani
-ocd-division/country:in/state:mh/district:pu,Pune
-ocd-division/country:in/state:mh/district:rg,Raigad
-ocd-division/country:in/state:mh/district:rt,Ratnagiri
-ocd-division/country:in/state:mh/district:sn,Sangli
-ocd-division/country:in/state:mh/district:st,Satara
-ocd-division/country:in/state:mh/district:si,Sindhudurg
-ocd-division/country:in/state:mh/district:so,Solapur
-ocd-division/country:in/state:mh/district:th,Thane
-ocd-division/country:in/state:mh/district:wr,Wardha
-ocd-division/country:in/state:mh/district:ws,Washim
-ocd-division/country:in/state:mh/district:yt,Yavatmal
+id,name
+ocd-division/country:in,India
+ocd-division/country:in/state:ap,Andhra Pradesh
+ocd-division/country:in/state:ap/district:an,Anantapur district; Andhra Pradesh

```

Comment:
Please put all districts in a districts.csv file.

Reply:
Done